helpers/cron-requester/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "chainId": "band-guanyu-devnet-2",
+  "endpoint": "http://guanyu-devnet.bandchain.org/rest",
+  "mnemonic": "final little loud vicious door hope differ lucky alpha morning clog oval milk repair off course indicate stumble remove nest position journey throw crane",
```

Comment:
Need to make sure this account always have a balance/always can make requests

Reply:
This is just example config for an illustrative purpose -- it's up to the users to make sure the account they're using has enough BAND.

![image](https://user-images.githubusercontent.com/891585/84598826-ad5bf100-ae97-11ea-88cf-705be1931cce.png)

Reply:
I added a warning though.